mmv1/products/bigqueryanalyticshub/api.yaml

```diff
@@ -92,3 +92,131 @@ objects:
+        name: listing_id
+        description: |-
+          The ID of the listing. Must contain only Unicode letters, numbers (0-9), underscores (_). Should not use characters that require URL-escaping, or characters outside of ASCII, spaces.
+        required: true
+        input: true
+        url_param_only: true   
+      - !ruby/object:Api::Type::String
+        name: "location"
+        description: |
+          The name of the location this data exchange listing.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: displayName
+        description: |-
+          Human-readable display name of the listing. The display name must contain only Unicode letters, numbers (0-9), underscores (_), dashes (-), spaces ( ), ampersands (&) and can't start or end with spaces.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: description 
+        description: |-
+          Short description of the listing. The description must not contain Unicode non-characters and C0 and C1 control codes except tabs (HT), new lines (LF), carriage returns (CR), and page breaks (FF). 
+      - !ruby/object:Api::Type::String
+        name: primaryContact
+        description: |-
+          Email or URL of the primary point of contact of the listing.
+      - !ruby/object:Api::Type::String
+        name: documentation
+        description: |-
+          Documentation describing the listing.
+      - !ruby/object:Api::Type::String
+        name: icon
+        description: |-
+          Base64 encoded image representing the listing.
+      - !ruby/object:Api::Type::String
+        name: requestAccess
+        description: |-
+          Email or URL of the request access of the listing. Subscribers can use this reference to request access.
+      - !ruby/object:Api::Type::NestedObject
+        name: dataProvider
+        description: Details of the data provider who owns the source data.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'name'
+            description: Name of the data provider.
+          - !ruby/object:Api::Type::String
+            name: 'primaryContact'
+            description: Email or URL of the data provider.
+      - !ruby/object:Api::Type::NestedObject
+        name: publisher
+        description: Details of the publisher who owns the listing and who can share the source data.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'name'
+            description: Name of the listing publisher.
+          - !ruby/object:Api::Type::String
+            name: 'primaryContact'
+            description: Email or URL of the listing publisher.
+      - !ruby/object:Api::Type::Array
+        name: 'categories'
+        description: |
+          Categories of the listing. Up to two categories are allowed.
+        item_type: !ruby/object:Api::Type::Enum
```

Comment:
We may want to just make this a type of String in case the categories ever change. We try to avoid enum when the set of values may change over time as an added field will be impossible to use with an older provider version

Reply:
fair enough

Reply:
done